docs/codeql/reusables/advanced-query-execution.rst

```diff
@@ -4,15 +4,15 @@
    <https://docs.github.com/en/code-security/codeql-cli/using-the-codeql-cli/using-custom-queries-with-the-codeql-cli#including-query-metadata>`__. You can also execute queries using the following
```

Comment:
Is this file still being used? I can't find any references to it in github/codeql.